person.rb

```diff
@@ -24,16 +24,13 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Done